packages/ckeditor5-dev-env/lib/release-tools/tasks/update-package-versions.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const chalk = require( 'chalk' );
+const { execSync } = require( 'child_process' );
+const fs = require( 'fs' );
+const glob = require( 'glob' );
+
+/**
+ * Updates `@ckeditor/ckeditor5-*` and `ckeditor5` dependencies in `packages/*` and `release/*`
+ * directories to the latest version. Changes in `packages/*` will be committed as well.
+ *
+ * See https://github.com/cksource/ckeditor5-internal/issues/1123
+ *
+ * @param {Object} options
+ * @param {String} options.cwd The root directory of the repository in which
+ * the script is being used.
+ * @param {Boolean} options.dryRun Prevents the script from committing, and
+ * instead shows list of files that was changed in `packages/*` directory.
+ * @param {Boolean} options.diff Instead of list of files, it displays detailed
+ * list of changes from each file. Has no effect without options.dryRun.
+ */
+module.exports = function updatePackageVersions( options ) {
+	const execOptions = {
+		stdio: 'inherit',
+		cwd: options.cwd
+	};
+
+	console.log( '\n📍 ' + chalk.blue( 'Updating CKEditor 5 dependencies...\n' ) );
+
+	try {
+		const packagesResult = updateDirectory( 'packages', options.cwd );
+		console.log( getFeedback( packagesResult ) );
+
+		const releaseResult = updateDirectory( 'release', options.cwd );
+		console.log( getFeedback( releaseResult ) );
+
+		if ( packagesResult.updated ) {
+			console.log( '\n📍 ' + chalk.blue( 'Committing the changes from \'packages/*\'...\n' ) );
+
+			if ( options.dryRun ) {
+				console.log( chalk.yellow( 'DRY RUN mode - no changes will be committed.' ) );
```

Comment:
I think it's fine like this, since it shows you what WOULD happen normally, but then `dry-run` "interrupts" the process, but if you feel strong about this, i can change this so that only the dry run message is displayed, and update it to say something along the lines of "Changes would now be commited, but this is dry run, so that step is skipped."

Reply:
As i think about it, You're right, i will update it